AUTHORS

```diff
@@ -1,8 +1,38 @@
-Vincent Garonne, CERN PH-ADP-CO, <vincent.garonne@cern.ch>, 2012-2014
-Mario Lassnig, CERN PH-ADP-CO, <mario.lassnig@cern.ch>, 2012-2014
-Angelos Molfetas, CERN PH-ADP-CO, <angelos.molfetas@cern.ch>, 2012
-Martin Barisits, CERN PH-ADP-CO, <martin.barisits@cern.ch>, 2012-2014
-Thomas Beermann, CERN PH-ADP-CO, <thomas.beermann@cern.ch>, 2012-2014
-Ralph Vigne, CERN PH-ADP-CO, <ralph.vigne@cern.ch>, 2012-2014
-Cedric Serfon, CERN PH-ADP-CO, <cedric.serfon@cern.ch>, 2013-2014
-Gancho Dimitrov, CERN PH-ADP-CO, <gancho.dimitrov@cern.ch>, 2013-2014
+Individual contributors to the source code:
+-------------------------------------------
+
+Mario Lassnig <mario.lassnig@cern.ch>, 2012-2017
+Vincent Garonne <vgaronne@gmail.com>, 2012-2017
+Angelos Molfetas <Angelos.Molfetas@cern.ch>, 2012-2012
+Martin Barisits <martin.barisits@cern.ch>, 2012-2017
+Thomas Beermann <thomas.beermann@cern.ch>, 2012-2017
+Ralph Vigne <ralph.vigne@cern.ch>, 2012-2016
+Graeme Stewart <graeme.andrew.stewart@cern.ch>, 2012-2012
+Yun-Pin Sun <winter0128@gmail.com>, 2012-2013
+Cedric Serfon <cedric.serfon@cern.ch>, 2012-2017
+Luis Rodrigues <lfrodrigues@gmail.com>, 2013-2013
+WeiJen Chang <e4523744@gmail.com>, 2013-2014
+Gancho Dimitrov <gancho.dimitrov@cern.ch>, 2013-2013
+Wen Guan <wguan.icedew@gmail.com>, 2014-2017
+David Cameron <d.g.cameron@gmail.com>, 2014-2016
+Tomáš Kouba <tomas.kouba@cern.ch>, 2014-2015
+Cheng-Hsi Chao <cheng-hsi.chao@cern.ch>, 2014-2014
+Evangelia Liotiri <evangelia.liotiri@cern.ch>, 2014-2015
+Joaquín Bogado <jbogado@linti.unlp.edu.ar>, 2014-2017
+Fernando López <fernando.e.lopez@gmail.com>, 2015-2016
+Sylvain Blunier <sylvain.blunier@cern.ch>, 2016-2016
+Tomas Javurek <tomas.javurek@cern.ch>, 2016-2017
+Brian Bockelman <bbockelm@cse.unl.edu>, 2016-2017
+Tobias Wegner <twegner@cern.ch>, 2017-2017
+Frank Berghaus <frank.berghaus@cern.ch>, 2017-2017
+Vitjan Zavrtanik <vitjan.zavrtanik@cern.ch>, 2017-2017
+Stefan Prenner <stefan.prenner@cern.ch>, 2017-2017
+Joaquin Bogado <jbogadog@cern.ch>, 2017-2017
+Nicolo Magini <Nicolo.Magini@cern.ch>, 2017-2017
+Oliver Freyermuth <o.freyermuth@googlemail.com>, 2017-2017
+
+Organisations employing contributors
+------------------------------------
+European Organisation for Nuclear Research (CERN)
+University of Oslo (Norway)
+
```

Comment:
Institutions for the 2017 contributors also include:
- University of Wisconsin Madison
- National University of La Plata
- Albert Ludwigs Universitaet Freiburg
- University of Nebraska Lincoln
- Bergische Universitaet Wuppertal
- University of Victoria
- INFN e Universita Genova
- University of Bonn

I didn't backtrack the other years, I don't think it's necessary. If it bothers somebody they can just add it themselves.

Reply:
thank. Universitaet  vs. Universität ?

Reply:
I just copy&pasted the names from the CERN DB. We can also go for the english ones, I have no preference really.